docs/learning/README.md

```diff
@@ -108,9 +143,20 @@ This is a general set of technical links and recommended reading our team has fo
   - [5P;1R - Bitcoin's Elliptic Curve Cryptography](https://olshansky.substack.com/p/5p1r-bitcoins-elliptic-curve-cryptography)
   - [5P;1R - Jellyfish Merkle Tree](https://olshansky.substack.com/p/5p1r-jellyfish-merkle-tree)
 
-#### General Blogs
+#### Blogs - General Subscription Suggestions
```

Comment:
Blogs prefix is unnecessary - also, this isn't a pattern you follow anywhere else